sql/template_elasticdl.go

```diff
@@ -142,15 +143,14 @@ class PredictionOutputsProcessor(BasePredictionOutputsProcessor):
                 "MAXCOMPUTE_PROJECT",
                 "MAXCOMPUTE_AK",
                 "MAXCOMPUTE_SK",
-                "MAXCOMPUTE_ENDPOINT",
             )
         ):
             self.odps_writer = ODPSWriter(
                 os.environ["MAXCOMPUTE_PROJECT"],
                 os.environ["MAXCOMPUTE_AK"],
                 os.environ["MAXCOMPUTE_SK"],
-                os.environ["MAXCOMPUTE_ENDPOINT"],
-                table = "{{.PredictOutputTable}}",
+                os.environ["MAXCOMPUTE_ENDPOINT"] if "MAXCOMPUTE_ENDPOINT" in os.environ else None,
```

Comment:
How about using  `os.environ.get('MAXCOMPUTE_ENDPOINT', None)` to get the default value,  and also applying to other the MaxCompute env?